docs/search_algorithm_switching.md

```diff
@@ -1,37 +1,533 @@
+        genre_unstem_search^15
+        siku_subject_unstem_search
+        local_subject_unstem_search
+        homoit_subject_unstem_search
+        cjk_subjec
+      </str>
+      <str name="subject_pf">
+        subject_topic_unstem_search^250
+        subject_unstem_search^200
+        genre_unstem_search^150
+        siku_subject_unstem_search^10
+        local_subject_unstem_search^10
+        homoit_subject_unstem_search^10
+        cjk_subject^10
+      </str>
+
+      <int name="ps">3</int>
+      <float name="tie">0.01</float>
+
+      <str name="fl">
+        id,
+        score,
+        author_display,
+        marc_relator_display,
+        format,
+        pub_created_display,
+        title_display,
+        title_vern_display,
+        isbn_s,
+        oclc_s,
+        lccn_s,
+        holdings_1display,
+        electronic_access_1display,
+        electronic_portfolio_s,
+        cataloged_tdt,
+        contained_in_s
+      </str>
+
+      <str name="facet">true</str>
+      <str name="facet.mincount">1</str>
+      <str name="facet.limit">10</str>
+      <str name="facet.field">format</str>
+      <str name="facet.field">language_facet</str>
+      <str name="facet.field">pub_date_start_sort</str>
+      <str name="facet.field">advanced_location_s</str>
+      <str name="f.format.facet.sort">index</str>
+      <str name="f.advanced_location_s.facet.sort">index</str>
+      <str name="f.language_facet.facet.limit">1000</str>
+      <str name="f.advanced_location_s.facet.limit">500</str>
+    </lst>
+  </initParams>
+```
+
+In order to further extend the default parameters sent within HTTP requests to
+Solr, one need not directly modify the `initParams`, but instead override by following this pattern:
+
+```xml
+    <lst name="appends">
+      <str name="boost">if(termfreq(text,'engineering'),100,0)</str>
+    </lst>
+```
+
+In this case, `<lst>` specifies the list of parameters to send in the request,
+and the attribute `appends` qualifies the the child elements are to be appended
+to the `<lst>` elements specified within the `initParams`. The `<str>` encodes a parameter used for boosting search results by certain criteria (hence the `name="boost"`). The function query `if(termfreq(text, 'engineering'),100,0)` determines if the `text` field contains the string `engineering`, and if so, boosts the weight of the result by 100. If it does not contain `engineering`, it boosts the weight by 0.For more information, please referencing the [Solr documentation specific to configuring request handlers](https://solr.apache.org/guide/solr/latest/configuration-guide/requesthandlers-searchcomponents.html#defaults-appends-and-invariants).
+
+#### Submitting Proposed Changes
+
+Please proceed at this point with opening a pull request and requesting a review
+of the updated `ConfigSet`. An example of this can be found within the
+following:
+
+- https://github.com/pulibrary/pul_solr/pull/375
+- https://github.com/pulibrary/pul_solr/pull/377
```

Comment:
Can we add href text for these two pull requests? 
[Adding a new example requestHandler for algorithm switching]
[Refactors the requestHandlers to utilize init params]